nix/scripts/kill_proc_prompt.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Function to prompt user for terminating a process
+kill_proc_prompt() {
```

Comment:
You know, if this is a script it should act as a script, not just a library. It should be called directly with positional argument. Might be useful for other scripts.

Reply:
resolved in https://github.com/status-im/status-mobile/pull/18656/commits/4d46f1450a6b1fb08f6cdc4dd345cd2d152acf7a